src/main/java/com/microsoft/sqlserver/jdbc/DataTypes.java

```diff
@@ -1184,7 +1201,8 @@ boolean convertsTo(JDBCType jdbcType) {
                 SSType.Category.CHARACTER,
                 SSType.Category.LONG_CHARACTER,
                 SSType.Category.NCHARACTER,
-                SSType.Category.LONG_NCHARACTER)),
+                SSType.Category.LONG_NCHARACTER,
+                SSType.Category.SQL_VARIANT)),
```

Comment:
same question as the last one. Just for my own understanding, why we only support updater conversion from Numeric to  SQL_VARIANT?